jit-service/src/main/java/org/dcsa/jit/mapping/VesselMapper.java

```diff
@@ -3,24 +3,40 @@
 import org.dcsa.jit.persistence.entity.Vessel;
 import org.dcsa.jit.transferobjects.TimestampVesselTO;
 import org.dcsa.jit.transferobjects.VesselTO;
+import org.dcsa.jit.transferobjects.enums.CarrierCodeListProvider;
+import org.mapstruct.AfterMapping;
 import org.mapstruct.Mapper;
 import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
 import org.mapstruct.Mappings;
 
 @Mapper(componentModel = "spring")
 public interface VesselMapper {
-  // FIXME @Mapping(target = "vesselOperatorCarrierCode", source = "vesselOperatorCarrier.") // smdgCode or nmftaCode
   VesselTO toTO(Vessel vessel);
 
   @Mapping(target = "isDummy", source = "isDummy", defaultValue = "false")
   Vessel toEntity(VesselTO vesselTO);
 
   @Mappings(value = {
-    @Mapping(target = "vesselName", source = "name"),
+    @Mapping(target = "name", source = "name"),
```

Comment:
Don't need to explicitly map fields of the same name

Reply:
Yeah, I oversaw that